### PR TITLE
feat: add support for dependabot

### PR DESCRIPTION
**Pull request by vinayakkulkarni**
- [x] When merged, this will enable support for `dependabot` as one of the devTools options

### CODE REVIEW

**Review by clarkdo — COMMENTED** [review]
As we're using [renovate](https://github.com/renovatebot/renovate) and also this is kind of outside scope of Nuxt.js, so I'm not sure if it's proper to be merged.

Let's hear feedback from @Atinux and @pi0

**Comment by vinayakkulkarni** [comment]
> As we're using renovate and also this is kind of outside scope of Nuxt.js, so I'm not sure if it's proper to be merged.
>
> Let's hear feedback from @Atinux and @pi0

This is basically for a layman user who wants to have `dependabot` support out-of-the-box (without the need of adding the `.github/dependabot.yml` file manually afterwards). This PR doesn't affect the `create-nuxt-app` repo or the nuxt ecosystem in any ways (which uses renovate)

File: packages/create-nuxt-app/lib/prompts.js [hunk 1]
```diff
@@ -109,10 +109,19 @@ module.exports = [
     type: 'checkbox',
     choices: [
       { name: 'jsconfig.json (Recommended for VS Code if you\'re not using typescript)', value: 'jsconfig.json' },
-      { name: 'Semantic Pull Requests', value: 'semantic-pull-requests' }
+      { name: 'Semantic Pull Requests', value: 'semantic-pull-requests' },
+      { name: 'Dependabot (For auto-updating dependencies)', value: 'dependabot' }
```
Comment (pi0) [inline packages/create-nuxt-app/lib/prompts.js:113]: Dependabot only supports github and is going to github built-in soon. While still major share for private repo usage is for gitlab. (https://github.com/dependabot/dependabot-core/issues/399). In that case, i'm more in favor or having [renovate](https://github.com/renovatebot/renovate) option first but this PR should be valid as while as we mention it is for **github only**

**Review by pi0 — CHANGES_REQUESTED** [review]
We should mention GH only or switch to renovate

File: packages/create-nuxt-app/lib/prompts.js (hunk 1 above)
Comment (vinayakkulkarni) [reply packages/create-nuxt-app/lib/prompts.js:113]: I most definitely can update the text to notify users that it'll work only on GitHub :)

Comment (vinayakkulkarni) [reply packages/create-nuxt-app/lib/prompts.js]: - [x] Fixed in 5cc555c

**Review by pi0 — APPROVED** [review]
:+1: Looks awesome thanks :green_heart:

**Review by clarkdo — APPROVED** [review]
Great, can you please update README.md as well ?

**Comment by vinayakkulkarni** [comment]
> Great, can you please update README.md as well ?

- [x] Done in a0da578

**Pull request merged**